apps/nextjs-website/src/app/solutions/[solutionSlug]/[...solutionSubPathSlugs]/page.tsx

```diff
@@ -28,24 +26,6 @@ type Params = {
   solutionSubPathSlugs: string[];
 };
 
```

Comment:
Like we have discuss in meeting i will keep these pages static for now

```suggestion
// TODO: remove when solutions metadata will be managed in strapi
export const revalidate = REVALIDATE_LONG_INTERVAL;

const SOLUTION_SLUG_PATH_INDEX = 2;
const SOLUTION_SUB_PATH_INDEX = 3;
export async function generateStaticParams(): Promise<Params[]> {
  const solutions = await getSolutionsMetadata();
  const solutionParams = solutions
    .map(({ path }) => path.split('/'))
    .filter((paths) => paths.length > SOLUTION_SUB_PATH_INDEX)
    .map((paths) => {
      return {
        solutionSlug: paths[SOLUTION_SLUG_PATH_INDEX],
        solutionSubPathSlugs: paths.slice(SOLUTION_SUB_PATH_INDEX),
      };
    });
  return solutionParams;
}

```